src/options.rs

```diff
@@ -185,6 +194,23 @@ pub fn builder_from_flags<I>(args: I)
         }
     }
 
+    if let Some(what_to_generate) = matches.value_of("generate") {
```

Comment:
It is probably better to state clearly what would happen if both `--generate` and one of `--ignore-*` options are specified. I guess it's probably easier to just reject that kind of combinations.